src/main/java/org/myrobotlab/swing/PythonGui.java

```diff
@@ -223,6 +223,10 @@ public EditorPanel addNewEditorPanel(Script script) {
 		scripts.put(script.getName(), panel);
 		return panel;
 	}
+	
+	public void onAppendScript(Script script) {

```

Comment:
This method will cause an infinite loop.   Please remove or fix.

Reply:
This method calls itself .. that's a recursive method and is most certain to crash.  This is dangerous code and should be removed.

Reply:
done & worky